packages/ts/src/core/container/index.ts

```diff
@@ -148,4 +150,10 @@ export class ContainerCore {
     this._resizeObserver?.disconnect()
     this.svg.remove()
   }
+
+  getAriaDescription (): string {
```

Comment:
`getAriaDescription` → `protected _getAriaDescription`

Reply:
I find it a bit difficult to understand how this method will work and I don't think it's going to work as supposed:

When you call `select(this._container).attr('aria-label', this.getAriaDescription())`, it'll first call the `getAriaDescription` method of the container instance, which in turn will call `super.getAriaDescription` that will return `config.ariaLabel` if provided. So you'll end up having the provided and generated descriptions at the same time.

Let's separate the description generation logic from the rendering logic. It will make it easier to understand the code.

Reply:
As per the updated logic, we generate description like so-
`const description = config.ariaAutoLabel ? this._getAriaDescription() : config.ariaLabel`